### PR TITLE
Add jobs for QE and Ion

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
They both need to test changes in sumaform.

### CODE REVIEW

**Review by nodeg — APPROVED** [review]
LGTM. Thank you for providing the pipeline. I think for the future this is very helpful.

File: jenkins_pipelines/environments/common/pipeline-pull-request-envs.groovy [hunk 1]
```diff
@@ -4,6 +4,20 @@ if (env.JOB_NAME == "uyuni-prs-ci-tests-jordi") {
     // if you change the sumaform repo or reference, you need to remove the sumaform directory from the results folder
     sumaform_gitrepo = "https://github.com/witekest/sumaform.git";
     sumaform_ref = "server_monitoring";
+} else if (env.JOB_NAME == "uyuni-prs-ci-tests-ion") {
+    first_env = 9;
+    last_env = 9;
+    // if you change the sumaform repo or reference, you need to remove the sumaform directory from the results folder
+    sumaform_gitrepo = "https://github.com/uyuni-project/sumaform.git";
+    sumaform_ref = "master";
```
Comment (SchoolGuy) [inline jenkins_pipelines/environments/common/pipeline-pull-request-envs.groovy:12]:
```suggestion
    sumaform_ref = "master-cobbler-3.3.1";
```

**Review by SchoolGuy — CHANGES_REQUESTED** [review]
Let's put the right branch name in here. :)

**Review by SchoolGuy — APPROVED** [review]
LGTM

**Pull request merged**